src/app_service/common/network_constants.nim

```diff
@@ -180,14 +180,31 @@ var NETWORKS* = %* [
     "blockExplorerUrl": "https://sepolia.etherscan.io/",
     "iconUrl": "network/Network=Ethereum",
     "chainColor": "#51D0F0",
-    "shortName": "sep",
+    "shortName": "eth",
     "nativeCurrencyName": "Ether",
     "nativeCurrencySymbol": "ETH",
     "nativeCurrencyDecimals": 18,
     "isTest":  true,
     "layer":   1,
     "enabled": true,
```

Comment:
@alaibe doesn't this need to be false?

Reply:
Nope, true is good. Because it needs to be enabled when sepolia is enabled